packages/dialtone-tokens/dialtone-transforms.js

```diff
@@ -85,6 +85,16 @@ export function registerDialtoneTransforms (styleDictionary) {
       return ['color'].includes(token.type);
     },
     transform: (token) => {
+      if (token.$extensions?.['studio.tokens']?.modify?.type === 'alpha') {
+        const alpha = parseFloat(token.$extensions['studio.tokens'].modify.value);
+        const alphaHex = Math.round(alpha * 255).toString(16).padStart(2, '0');
```

Comment:
I'm pretty sure you could just simply tack alphaHex on to the end of token.value and everything else would just work right?

Example:

```
transform: (token) => {
  let alphaHex = '';
  if (token.$extensions?.['studio.tokens']?.modify?.type === 'alpha') {
    const alpha = parseFloat(token.$extensions['studio.tokens'].modify.value);
    alphaHex = Math.round(alpha * 255).toString(16).padStart(2, '0');
  }
  const hex8 = Color(token.value + alphaHex).toHex8();
  return `Color(0x${hex8.slice(6) + hex8.slice(0, 6)})`;
},
```

Reply:
If token.value contains 8 characters already, might need to replace the final two characters instead of append.

Reply:
That won't work because for the token with `$extensions` the value comes in Flutter format already, i.e `Color(0xffabcd)`, so you can't append the alphaHex.

Reply:
Alright, we can merge this then.